channel/consensus_channel/consensus_channel_test.go

```diff
@@ -7,11 +7,49 @@ import (
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/google/go-cmp/cmp"
+
 	"github.com/statechannels/go-nitro/channel/state"
-	"github.com/statechannels/go-nitro/internal/testdata"
 	"github.com/statechannels/go-nitro/types"
 )
 
+type actor struct {
```

Comment:
These definitions should probably live in `helpers_test.go`

Reply:
Done in another PR: db3eeec081d65f24cbce3106108692186b3dd8fc